Library/Homebrew/brew.sh

```diff
@@ -102,6 +102,7 @@ then
   if [[ "$HOMEBREW_MACOS_VERSION_NUMERIC" -lt "100900" ]]
   then
     HOMEBREW_SYSTEM_GIT_TOO_OLD="1"
```

Comment:
Is this still used now?

Reply:
I hope so. There's no other check (that I've seen) for too-old-ness. Without that, it's just a manual option.
Maybe there's a better or additional check, though? Parsing `git --version`?

Reply:
Yes, parsing `git --version` would be preferable, at least on Linux.
@reitermarkus Which do you prefer on macOS? Checking the version of Git, or checking the version of macOS?

Reply:
I actually just meant if the variable `HOMEBREW_SYSTEM_GIT_TOO_OLD` is still used, not the check itself.

Reply:
It seems that `HOMEBREW_SYSTEM_CURL_TOO_OLD` is redundant, because `HOMEBREW_FORCE_SYSTEM_CURL` is set at the same time. The `GIT` variables are now in the same situation.
I guess the question is, is there a case for using an auto-determined `TOO_OLD` separately from a user-settable `FORCE`?

Reply:
`HOMEBREW_SYSTEM_GIT_TOO_OLD` does not appear to be used.

`HOMEBREW_SYSTEM_CURL_TOO_OLD` is used here:
https://github.com/Homebrew/brew/blob/af9ddfe899c3593a4d8168d4ade61eccd01d19b0/Library/Homebrew/extend/os/mac/development_tools.rb#L48